R/ggplot.R

```diff
@@ -1247,7 +1247,7 @@ autoplot.mts <- function(object, colour=TRUE, facets=FALSE, ...){
     
     #Initialise ggplot object
     mapping <- ggplot2::aes_(y=~y, x=~x, group=~series)
-    if(colour){
+    if (colour & (!facets | !missing(colour))){
```

Comment:
Not sure that it makes sense to have colour and !missing(colour) in the one if statement.